server/bleep/src/ee/webserver.rs

```diff
@@ -0,0 +1,36 @@
+use axum::{
+    extract::{Query, State},
+    Json,
+};
+use serde::Deserialize;
+
+use crate::{
+    repo::BranchFilter,
+    webserver::{
+        prelude::*,
+        repos::{RepoParams, ReposResponse},
+    },
+    Application,
+};
+
+#[derive(Deserialize)]
+pub(crate) struct RepositoryPatch {
+    branch_filter: BranchFilter,
+}
+
+/// Patch a repository with the given payload
+/// This will automatically trigger a sync
+//
+pub(crate) async fn patch_indexed(
```

Comment:
Could this be used for deleting as well as syncing branches? We should rename to something branch specific like `sync_branch` or `patch_branch` (if deletes might be supported).